src/component/handlers/edit/editOnCopy.js

```diff
@@ -31,7 +33,31 @@ function editOnCopy(editor: DraftEditor, e: SyntheticClipboardEvent<>): void {
     return;
   }
 
-  editor.setClipboard(getFragmentFromSelection(editor._latestEditorState));
+  const fragment = getFragmentFromSelection(editor._latestEditorState);
+
+  editor.setClipboard(fragment);
+
+  // IE11 does not support ClipboardEvent.clipboardData.
+  if (e.clipboardData && fragment) {
```

Comment:
I think skipping IE11 here is fair, since it doesn't handle HTML in the clipboard to start with.

Reply:
We handle IE11 in our fork in a rather different way. We have two hidden divs, one for copy and one for paste, that exist to accept or receive incoming content. In the copy case we copy the HTML of the component into this div and temporarily move selection so that the browser is able to copy that content in the way it wants to. For paste we temporarily move focus to the other div and then grab the content from there and convert it into a fragment.

The trick is ensuring those hidden divs are styled the same as the editor and that the focus games play out correctly but it works extremely well.

Reply:
@colinjeanne could you share your code's fork, at least the part that's relevant to this? I think I'd be keen to add this to my editor as well, depending on how much code it takes.

Reply:
Yeah, there are a few different changes, the first was https://github.com/textioHQ/draft-js/pull/21 followed later by https://github.com/textioHQ/draft-js/pull/26. We originally wrote this solution in order to get HTML into the clipboard even for browsers that didn't support adding it explicitly.